# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/libwebrtcglue/WebrtcEnvironmentWrapper.cpp
# Commit: 7a33c8b73972
# Full Hash: 7a33c8b7397200d89a0d3904669777e60550c445
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2025-06-11 18:38:51
# Regressor Bug: 1968812
# File Overlap Count: 1
# Description:
#   Bug 1968812 - Remove the CallWorker thread and use TaskQueue::Observer instead. r=bwc
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D252686
# ==============================================================================

diff -r a3ad3abc4d02 -r 7a33c8b73972 dom/media/webrtc/libwebrtcglue/WebrtcEnvironmentWrapper.cpp
--- a/dom/media/webrtc/libwebrtcglue/WebrtcEnvironmentWrapper.cpp	Wed Jun 11 13:37:22 2025 +0000
+++ b/dom/media/webrtc/libwebrtcglue/WebrtcEnvironmentWrapper.cpp	Wed Jun 11 13:37:23 2025 +0000
@@ -16,8 +16,7 @@
 /* static */ RefPtr<WebrtcEnvironmentWrapper> WebrtcEnvironmentWrapper::Create(
     const dom::RTCStatsTimestampMaker& aTimestampMaker) {
   RefPtr<WebrtcEnvironmentWrapper> wrapper = new WebrtcEnvironmentWrapper(
-      MakeUnique<webrtc::RtcEventLogNull>(),
-      MakeUnique<SharedThreadPoolWebRtcTaskQueueFactory>(),
+      MakeUnique<webrtc::RtcEventLogNull>(), CreateWebrtcTaskQueueFactory(),
       WrapUnique(new webrtc::MozTrialsConfig()), aTimestampMaker);
 
   return wrapper;